athlinks-schema.ttl

```diff
@@ -145,10 +161,48 @@ athlinks:bib
 
 athlinks:pace
   a owl:DataTypeProperty ;
-  rdfs:range schema:DateTime .
+  rdfs:range schema:Duration .
 
 athlinks:seriesType
   a owl:DataTypeProperty;
-  rdfs:range[ a xsd:string ;
-		owl:oneOf("Master", "Social", "Points", "Series")
-	     ] .
+  rdfs:range [ a xsd:string ;
+								owl:oneOf("Master", "Social", "Points", "Series")
+	     				] .
+
+# For  AthleticRace PROPERTIES
+athlinks:giveaway
+  a owl:DataTypeProperty;
+  rdfs:range xsd:string .
+
+athlinks:require_phone
+  a owl:DataTypeProperty;
+  rdfs:range xsd:boolean .
+
+athlinks:require_dob
+  a owl:DataTypeProperty;
+  rdfs:range xsd:boolean .
+
+athlinks:volunteer
+  a owl:DataTypeProperty;
+  rdfs:range xsd:boolean .
+
+For AthleticEvent PROPERTIES
+athlinks:details
+  a owl:DataTypeProperty;
+  rdfs:range xsd:string .
+
+athlinks:description
+  a owl:DataTypeProperty;
+  rdfs:range xsd:string .
+
+athlinks:isPartnerRace
+  a owl:DataTypeProperty;
+  rdfs:range xsd:boolean .
+
+athlinks:externalResultsUrl
```

Comment:
If we want our links to get this specific.. they should be extending MainEntityOfPage. But even in that case.. you could achieve the same thing, but using MainEnityOfPage for the externalraceUrl. and then on the "results' object use mainEntityOfpage to get the externalResultsUrl.